app/commands/concerns/decidim/extra_user_fields/create_registrations_commands_overrides.rb

```diff
@@ -35,9 +53,23 @@ def extended_data
           date_of_birth: form.date_of_birth,
           gender: form.gender,
           phone_number: form.phone_number,
-          location: form.location
+          location: form.location,
+          underage: form.underage,
+          statutory_representative_email: form.statutory_representative_email
         )
       end
+
+      def send_email_to_statutory_representative
+        return if form.statutory_representative_email.blank?
+
+        Decidim::ExtraUserFields::StatutoryRepresentativeMailer.inform(@user).deliver_now
```

Comment:
To be homogeneous with `app/commands/concerns/decidim/extra_user_fields/omniauth_commands_overrides.rb:90` it should be 

```suggestion
        Decidim::ExtraUserFields::StatutoryRepresentativeMailer.inform(@user).deliver_later
```